test/lib/test_tasks.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+# Copyright (c) 2016 EMC Corporation.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from __future__ import unicode_literals
+
+from unittest import TestCase
+import tempfile
+from hamcrest import assert_that
+
+from storops.lib import tasks
+from test.vnx.cli_mock import patch_cli, t_vnx
+import time
+
+
+class TestPQueue(TestCase):
+
+    def setUp(self):
+        self.path = tempfile.mkdtemp(prefix='/tmp/storops_')
+
+    def _get_queue(self, path):
+        return tasks.PQueue(path)
+
+    def test_queue_path(self):
+        q = self._get_queue(self.path)
```

Comment:
This line is repeated in each case.  You could put it in `setUp`.
And you could consider putting `q.stop()` in `tearDown`